filters/consistenthash/key.go

```diff
@@ -0,0 +1,33 @@
+package consistenthash
+
+import (
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/loadbalancer"
+)
+
+type consistentHashKey struct {
+	template *eskip.Template
+}
+
+func NewConsistentHashKey() filters.Spec { return &consistentHashKey{} }
```

Comment:
`NewConsistentHashKey` missing godoc? The description from the reference could be adapated here.